Create a library for parsing the logs from the SDK and example apps (#511)
ably/ably-asset-tracking-swift: Tools/LogParser/README.md

# LogParser

> A package for parsing the log output of the Ably Asset Tracking SDK and its example apps.

Reviewer: Might be worth being more specific here, as I'm assuming this isn't compatible with Android SDK logs:

```suggestion
A package for parsing the log output of the Ably Asset Tracking Swift SDKs and their example apps.
```

Author: Fair point, updated it 👍
